miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniApp.kt

```diff
@@ -40,25 +47,59 @@ abstract class MiniApp internal constructor() {
 
     companion object {
         private lateinit var instance: MiniApp
+        private lateinit var defaultSettings: MiniAppSdkConfig
 
         /**
-         * Instance of [MiniApp].
+         * Instance of [MiniApp] with custom [MiniAppSdkConfig].
+         * This function should only be used if you wish to use the SDK with custom settings
+         * which are changed at runtime for QA purposes, etc.
+         * Note that the default [instance] without [settings] uses the config settings from AndroidManifest.xml.
          *
          * @return [MiniApp] instance
          */
         @JvmStatic
-        fun instance(): MiniApp = instance
+        fun instance(settings: MiniAppSdkConfig = defaultSettings): MiniApp =
```

Comment:
Need to update the documentation for the method.

Reply:
Actually, I already edited it so could you read it again or suggest the content 🙇